packages/babel-plugin-transform-destructuring/test/fixtures/destructuring/destructuring-empty-in-for-body/input.js

```diff
@@ -0,0 +1 @@
+for ( let x in y ) for ( ; ; ) var { } = x ;
```

Comment:
We can test even more:
```js
for ( let x in y ) for ( ; { } = x ; {} = x ) var { } = x ;
```
So it will be empty destructuring in every children of ForStatement except `init`.